docs/references/troubleshooting.rst

```diff
@@ -45,27 +45,27 @@ Python package "xlrd"
 
 On **Windows** there can be issues installing xlrd. This could solve your troubles:
 
-1. Delete xlrd from requirements.txt
-2. Download the xlrd-1.2.0-py2.py3-none-any.whl file from `here <https://pypi.org/project/xlrd/#files>`__.
+1. Delete :code:`xlrd` from requirements.txt
+2. Download the :code:`xlrd-1.2.0-py2.py3-none-any.whl` file from `here <https://pypi.org/project/xlrd/#files>`__.
 3. Copy the file to main directory of the project on your laptop
-4. Install it manually writing pip install xlrd-1.2.0-py2.py3-none-any.whl
+4. Install it manually writing :code:`pip install xlrd-1.2.0-py2.py3-none-any.whl`
 
 Python package "wkhtmltopdf"
 ############################
 
-There can be issues installing ´wkhtmltopdf´. Solution can be found on the `packages documentation <https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf>`__.
+There can be issues installing :code:`wkhtmltopdf`. Solution can be found on the `packages documentation <https://github.com/JazzCore/python-pdfkit/wiki/Installing-wkhtmltopdf>`__.
 
 cbc-solver
 ##########
 
 While with Ubuntu the installation of the cbc solver should work rather well, even when adding it to the environment variables (like described in the installation instructions) can sometimes not work on Windows. This was experienced with Windows 10.
 
-A workaround is to directly put the `cbc.exe` file into the pycharm project, ie. in the same folder where also the CHANGELOG.md is located. Python/Oemof/Pyomo then are able to find the solver.
+A workaround is to directly put the :code:`cbc.exe` file into the pycharm project, ie. in the same folder where also the :code:`CHANGELOG.md` is located. Python/Oemof/Pyomo then are able to find the solver.
```

Comment:
```suggestion
A workaround is to directly put the :code:`cbc.exe` file into the root of the MVS repository, ie. in the same folder where also the :code:`CHANGELOG.md` file is located. Python/Oemof/Pyomo then are able to find the solver.
```
This workaround does not work for people who cloned the repo locally, not for people who pip installed it